File: src/main/java/org/jukeboxmc/item/ItemIronDoor.java
package org.jukeboxmc.item;

import org.jukeboxmc.block.BlockIronDoor;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ItemIronDoor extends Item {

    public ItemIronDoor() {
        super( 370 );
    }

    @Override
    public BlockIronDoor getBlock() {
        return new BlockIronDoor();
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/behavior/ItemIronDoor.kt
package org.jukeboxmc.server.item.behavior

import org.jukeboxmc.api.block.Block
import org.jukeboxmc.api.block.BlockType
import org.jukeboxmc.api.item.ItemType
import org.jukeboxmc.server.item.JukeboxItem

class ItemIronDoor(itemType: ItemType, countNetworkId: Boolean) : JukeboxItem(itemType, countNetworkId) {

    override fun toBlock(): Block {
        return Block.create(BlockType.valueOf(this.getType().name))
    }

}